sdk/core/Azure.Core.TestFramework/src/ClientTestFixtureAttribute.cs

```diff
@@ -18,18 +18,18 @@ public class ClientTestFixtureAttribute : NUnitAttribute, IFixtureBuilder2, IPre
     {
         private readonly struct TestVersion : IComparable
         {
-            public IComparable Comparable { get; }
+            private readonly IComparable _comparable;
```

Comment:
Still don't see why we need this at all, as compared to just storing in Object.

Reply:
Object is the original thing that was passed in which is needed on line 170 (used to be 169 in a previous comment but it moved).  _comparable is the "converted thing" that can be used to compare for finding the Min / Max which is of a completely different type.

If I don't keep both in this struct then I will need some sort of dictionary to look up the original object from the comparable after the Max method is called.

Reply:
Ah, got it.